pkg/apis/kubermatic/v1/cluster_templates.go

```diff
@@ -71,6 +75,7 @@ type ClusterTemplateList struct {
 	metav1.TypeMeta `json:",inline"`
 	metav1.ListMeta `json:"metadata,omitempty"`
 
+	// Refers to the list of the ClusterTemplate ojbects.
```

Comment:
Comments should always start with the field its referring to, so it would be 

```suggestion
	// Items refers to the list of the ClusterTemplate objects.
```